### ISSUE TITLE
[Dashboard] Decoupling dashboard and dashboard lifetime from Ray Cluster 

### ISSUE THREAD

**Issue by Superskyyy** [ray-project/ray#46444]
### Description

With Ray starting to support the virtual cluster (vCluster) concept and we are seeing advanced multi-cluster per user setups, the Ray dashboard components should not be bound to a single Ray cluster's lifetime anymore, since it makes multi-tenancy sharing and telemetry data persistence complex to implement. Plus that the dashboard would go down together if the head node goes down (fate-sharing), making it difficult to backtrack what happened (and what was executing) during a major incident. 

### Use case

Doing so will bring below benefits:
1. Dashboard can optionally read from a persistence history server (observability database) instead of pulling directly from a running GCS. (GCS/HA redis writes to persistence store)
2. Dashboard side overhead will not accidentally bring down the head node.
3. Users can attach their own external monitoring platforms same way as job dashboard, to manage large amount of clusters.
4. Each user gets their dashboard, which can be multi physical cluster or vclusters.
5. Allow checking dashboard even after a cluster was preempted/shutdown.

**Comment by Bye-legumes** [comment]
similar issue https://github.com/ray-project/ray/issues/45940

**Comment by yucai** [comment]
Share we have a abstraction layer in front of DataBase? So that, different DB solution can be used.
@anyscalesam , @Bye-legumes I heard ByteDance had some solution already, kindly share with me if you have. Thanks a lot!

**Comment by anyscalesam** [comment]
let's grab time to chat more about this cc @alanwguo

**Comment by jhasm** [comment]
@anyscalesam just checking if there is any update on this issue or the "export api"? 
Without external dashboard view all the ray dashboard features have very limited use for us. Thanks!

**Comment by nikitavemuri** [comment]
@jhasm Yes, this is an area we’re actively working on. We previously implemented an alpha version of the export API but observed regressions with large workloads, so this feature was redesigned to be more performant. We expect to release task state and metadata events this quarter, which can be used to persist the task table view, and will be following up with additional Ray resource types (eg: actors, nodes, jobs).

**Comment by 850552586** [comment]
@nikitavemuri I'd like to ask, when is the release date for this feature expected to be?

**Comment by jhasm** [comment]
Thanks @nikitavemuri , any way to see/track progress on the work? It will help us plan our own work around this feature.

**Comment by mucheng2035** [comment]
Is this function suspended?
